extra/3-magic-8-ball.js

```diff
@@ -101,7 +114,9 @@ test("magic 8 ball returns different values each time", () => {
     );
   }
 
-  let seenPositivities = new Set(Array.from(seenAnswers.values()).map(checkAnswer));
+  let seenPositivities = new Set(
+    Array.from(seenAnswers.values()).map(checkAnswer)
+  );
```

Comment:
Congratulations on your successfully completion of 2 extra exercises! May be we can talk about 2 dimensional array (2D) array and that will help you to complete this exercise. Cheers!